src/interpreter/runtime/values/i64.js

```diff
@@ -1,14 +1,12 @@
 // @flow
-
+const Long = require('long');
 
 const bits = 64;
 const type = 'i64';
 
-export function createValue(value: number): StackLocal {
-  value = (value | 0) % Math.pow(2, bits);
-
+export function createValue(value: Long): StackLocal {
   return {
     type,
-    value,
+    value: new Long(value.lower, value.upper)
```

Comment:
It's not a big deal but I like trailing commas (because you can duplicate a line more easily). Might be worth adding an ESLint/Prettier to keep our code consistent.

Reply:
definitely +1 to the idea of adding prettier, I was going to ask about what code formatter / linter you are using locally :-)

Reply:
on a related note, I guess you are using Flow in an ad-hoc fashion at the moment? I'm seeing quite a lot of Flow errors when running `make lint`

Reply:
Yes, that's absolutely my bad, I will have to fix them. For convenience (and to be consistent in my other JS projects) I configured ESLint to call Flow, I only have an integration with ESLint in my editor.

Reply:
🎉 🎉  Prettier, yeess 🎉 🎉